iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingView.java

```diff
@@ -693,17 +693,17 @@ public LineAdder newLine() {
 
     @Override
     public Iterable<Line> getLines() {
-        return Collections.unmodifiableCollection(index.getLines());
+        return index.getLines();
     }
 
     @Override
     public Stream<Line> getLineStream() {
-        return index.getLines().stream();
+        return index.getLineStream();
     }
 
     @Override
     public int getLineCount() {
-        return index.getLines().size();
+        return (int) index.getLineStream().count();
```

Comment:
Is it the optimize way to implement this? Could we simply sum the lineCount of the underlying networks and add the number of mergedLine? I don't know if it's really better, but it avoids to iterate over all the lines